packages/appstore/src/components/appstore-banner/svg-migration-banner.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Button, Text } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import AppstoreBannerImage from 'Assets/svgs/banner';
+
+const SVGMigrationBanner = observer(() => {
+    const {
+        ui: { is_mobile },
+    } = useStore();
```

Comment:
shall we follow the current standard way of destructuring?
```
const {ui} = useStore();
const {is_mobile} = ui;
```
this looks cute, right? 👗